src/components/Content/TaskHierarchy/Task/Task.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import * as postgrest from '../../../../api/postgrest';
+import TiPlus from 'react-icons/lib/ti/plus';
+import TiMinus from 'react-icons/lib/ti/minus';
+import FaHandODown from 'react-icons/lib/fa/hand-o-down';
+import FaCommentsO from 'react-icons/lib/fa/comments-o';
+import TaskForm from '../../TaskManager/TaskForm/TaskForm';
+import './Task.css';
+
+class Task extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showChildren: true,
+      showTaskForm: false
+    };
+  }
+
+  toggleChildren = () => {
+    this.setState({
+      ...this.state,
+      showChildren: !this.state.showChildren
+    });
+  }
+
+  toggleNewForm = () => {
+    this.setState({
+      ...this.state,
+      showTaskForm: !this.state.showTaskForm
+    });
+    // TODO: Post route for nested form
+  }
+
+  redirectToDiscuss = () => {
+    // this.props.history.push('/pursuance/1/discuss');
```

Comment:
@Moe-Shoman @MartyTheeMartian Let's fix `react-router` crap right after the merge